pkg/localstore/migration_kademlia_metrics.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package localstore
+
+import (
+	"time"
+)
+
+// DBSchemaKademliaMetrics is the bee schema identifier for kademlia metrics.
+const DBSchemaKademliaMetrics = "kademlia-metrics"
+
+// migrateKademliaMetrics removes all old existing
+// kademlia metrics database content.
+func migrateKademliaMetrics(db *DB) error {
```

Comment:
this is in the wrong package.... the migration should be in the statestore package, not localstore, as the statestore stores the kademlia metrics, not localstore